crates/shrs_core/src/builtin/debug.rs

```diff
@@ -28,20 +31,25 @@ impl BuiltinCmd for DebugBuiltin {
         ctx: &mut Context,
         rt: &mut Runtime,
         args: &Vec<String>,
-    ) -> anyhow::Result<BuiltinStatus> {
+    ) -> anyhow::Result<CmdOutput> {
+        let mut out = String::new();
         let cli = Cli::try_parse_from(args)?;
 
         match &cli.command {
             None => {
-                println!("debug utility");
+                let s = "debug utility\n";
```

Comment:
using "\n" may not work on all platforms (iirc windows wants "\r" and macos wants "\r\n" for example). this is not very much of an issue for now but maybe make a little note here

Reply:
Noted, will default to \r\n for now